android/libraries/rib-base/src/main/java/com/badoo/ribs/android/recyclerview/RecyclerViewHost.kt

```diff
@@ -0,0 +1,53 @@
+package com.badoo.ribs.android.recyclerview
+
+import android.content.Context
+import android.os.Parcelable
+import android.widget.FrameLayout
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.badoo.ribs.android.recyclerview.client.RecyclerViewRibResolver
+import com.badoo.ribs.core.Rib
+import io.reactivex.ObservableSource
+import kotlinx.android.parcel.Parcelize
+
+/**
+ * Considered experimental. Handle with care.
+ */
+interface RecyclerViewHost : Rib {
+
+    interface Dependency<T : Parcelable> {
+        fun hostingStrategy(): HostingStrategy
+        fun initialElements(): List<T>
+        fun recyclerViewHostInput(): ObservableSource<Input<T>>
+        fun resolver(): RecyclerViewRibResolver<T>
+        fun layoutManagerFactory(): LayoutManagerFactory
+        fun viewHolderLayoutParams(): FrameLayout.LayoutParams
```

Comment:
May be better to move it to `Customization`,  and provide default value, because almost in 99 cases you will use Match/Wrap